src/left-nav.jsx

```diff
@@ -265,7 +265,7 @@ const LeftNav = React.createClass({
         transform: `translate3d(${x}px, 0, 0)`,
         transition: !this.state.swiping && Transitions.easeOut(),
         backgroundColor: theme.color,
-        overflow: 'hidden',
+        overflow: 'auto',
```

Comment:
@oliviertassinari I think its fine. I don't see why someone would want to make their overflow hide.